### ISSUE TITLE
Files to move to uwlswd_scripts

### ISSUE THREAD

**Issue by briesenberg07** [UniverityOfWashingtonLibrariesSchemas/schemasProject#21]
@gerontakos 

I'll propose the following "starter set" of scripts for the uwlswd_scripts repo (not yet created).

- [rdf2rdfa-agent.xsl](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/blob/master/xslt/rdf2rdfa-agent.xsl)
- [rdf2rdfa-aggregation.xsl](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/blob/master/xslt/rdf2rdfa-aggregation.xsl)
- [rdf2rdfa-childLitCategories.xsl](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/blob/master/xslt/rdf2rdfa-childLitCategories.xsl)
- [rdf2rdfa-classes.xsl](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/blob/master/xslt/rdf2rdfa-classes.xsl)
- [rdf2rdfa-collection.xsl](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/blob/master/xslt/rdf2rdfa-collection.xsl)
- [rdf2rdfa-platform.xsl](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/blob/master/xslt/rdf2rdfa-platform.xsl)
- [rdf2rdfa-platformVocab.xsl](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/blob/master/xslt/rdf2rdfa-platformVocab.xsl)
- [rdf2rdfa-properties.xsl](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/blob/master/xslt/rdf2rdfa-properties.xsl)
- [rdf2rdfa-rdaExtension.xsl](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/blob/master/xslt/rdf2rdfa-rdaExtension.xsl)
- [rdf2rdfa-reconVovab.xsl](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/blob/master/xslt/rdf2rdfa-reconVovab.xsl)
- [rdf2rdfa-rightsStatement.xsl](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/blob/master/xslt/rdf2rdfa-rightsStatement.xsl)
- [rdf2rdfa-sourceResource.xsl](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/blob/master/xslt/rdf2rdfa-sourceResource.xsl)
- [rdf2rdfa-table.xsl](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/blob/master/xslt/rdf2rdfa-table.xsl)
- [rdf2rdfa-uwRdaExt.xsl](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/blob/master/xslt/rdf2rdfa-uwRdaExt.xsl)
- [rdf2rdfa-uwSemWeb.xsl](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/blob/master/xslt/rdf2rdfa-uwSemWeb.xsl)
- [rdf2rdfa-webResource.xsl](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/blob/master/xslt/rdf2rdfa-webResource.xsl)
- [schemaOrgMarkup-uwSemWeb.xsl](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/blob/master/xslt/schemaOrgMarkup-uwSemWeb.xsl)
- [schemaOrgMarkup.xsl](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/blob/master/xslt/schemaOrgMarkup.xsl)

I can move to the new repo and remove here once we confirm.

**Comment by gerontakos** [comment]
Yes all those files look right. I'm glad you remembered the schemaOrg stuff (I forgot).

**Comment by briesenberg07** [comment]
@gerontakos 
Whoops missed (at least) one:
- [cdm-2-rdf-ayp.xsl](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/blob/master/xslt/cdm-2-rdf-ayp.xsl)
